app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -162,65 +172,65 @@ object FullResultsPageHelper extends ViewHelper {
 
     val originalCompanyMsg = messages("fullResultsPage.oneOriginalCompany")
 
-    val upperLimitMsg = messages("fullResultsPage.upperLimit")
-
     val pointOneCompaniesCalcText = s"($associatedCompaniesText + $originalCompanyMsg)"
 
     val fraction = {
       val f = DecimalToFractionUtils.toFraction(yearConfig.marginalReliefFraction)
       f.numerator + " ÷ " + f.denominator
     }
-    val adjustedUpperLimit = marginalRate.adjustedUpperThreshold
 
     val taxableProfitIncludingDistributions = marginalRate.adjustedAugmentedProfit
 
     val daysInYear = Year.of(marginalRate.year).length()
 
-    govukTable(
-      Table(
-        rows = Seq(
-          Seq(
-            boldRow("1"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.adjustUpperLimit"))),
-            TableRow(content =
-              Text(s"$upperLimit $upperLimitMsg × ($daysString ÷ $daysInYear $daysMsg) ÷ $pointOneCompaniesCalcText")
-            ),
-            TableRow(content = Text(CurrencyUtils.format(adjustedUpperLimit)))
-          ),
-          Seq(
-            boldRow("2"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.taxableProfit"))),
-            TableRow(content = Text(s"${CurrencyUtils.format(taxableProfit)} × ($daysString ÷ $daysInYear $daysMsg)")),
-            TableRow(content = Text(CurrencyUtils.format(marginalRate.adjustedProfit)))
-          ),
-          Seq(
-            boldRow("3"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.taxableProfitDistributions"))),
-            TableRow(content =
-              Text(
-                s"${CurrencyUtils.format(marginalRate.adjustedProfit)} + ${CurrencyUtils
-                    .format(distributions)} × ($daysString ÷ $daysInYear $daysMsg)"
-              )
-            ),
-            TableRow(content = Text(CurrencyUtils.format(taxableProfitIncludingDistributions)))
-          ),
-          Seq(
-            boldRow("4"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.marginalReliefFraction"))),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.marginalReliefFraction.description"))),
-            TableRow(content = Text(fraction))
-          ),
-          Seq(
-            boldRow("5"),
-            TableRow(content = Text(messages("fullResultsPage.financialYear.fullCalculation"))),
-            TableRow(content = Text(s"""(${CurrencyUtils.format(adjustedUpperLimit)} - ${CurrencyUtils.format(
-                taxableProfitIncludingDistributions
-              )}) × (${CurrencyUtils.format(
-                marginalRate.adjustedProfit
-              )} ÷ ${CurrencyUtils.format(taxableProfitIncludingDistributions)}) × ($fraction)""")),
-            TableRow(content = Text(CurrencyUtils.format(ResultsPageHelper.marginalRelief(marginalRate))))
+    val isUpperLowerLimit = taxableProfitIncludingDistributions >= marginalRate.adjustedLowerThreshold
```

Comment:
isProfitsAboveLowerThreshold?

Also, should this be >= or >

Reply:
After re reading the ticket I think it should be > yes.